README.md

```diff
@@ -1,5 +1,8 @@
 # npg_ranger project
 
+[![Build Status](https://travis-ci.org/wtsi-npg/npg_ranger.svg?branch=master)](https://travis-ci.org/wtsi-npg/npg_ranger)
+[![NPM dependencies](https://david-dm.org/wtsi-npg/npg_ranger.svg)](https://david-dm.org/wtsi-npg/npg_ranger)
```

Comment:
I generated it with the online tool... it comes out as:

https://travis-ci.org/wtsi-npg/npg_ranger.svg?branch=master

I think the difference may be that I asked for an specific branch, otherwise it will show the last build.